pkg/skaffold/build/buildpacks/init.go

```diff
@@ -71,12 +71,30 @@ func (c ArtifactConfig) Path() string {
 
 // validate checks if a file is a valid Buildpack configuration.
 func validate(path string) bool {
+	valid := false
+
 	file := filepath.Base(path)
-	if file == "package.json" || file == "go.mod" || file == "project.toml" {
-		return !hasParent(path, "node_modules") && !hasParent(path, "vendor")
+	switch {
+	// Buildpacks project descriptor.
+	case file == "project.toml":
+		valid = true
+
+	// NodeJS.
+	case file == "package.json":
+		valid = true
+
+	// Go.
+	case file == "go.mod":
+		valid = true
+
+	// Python.
+	case file == "requirements.txt":
+		if _, err := os.Stat(filepath.Join(filepath.Dir(path), "Procfile")); err == nil {
+			valid = true
+		}
```

Comment:
The boolean return doesn't quite capture enough.  We should be able to return output a message here along the lines of "...$path seems to be a Python project, but Buildpacks requires a Procfile".  Or maybe this framework needs the ability to register a follow-up action (`REQUIRES_PROCFILE`) so that the CLI/UI/IDE can offer the user some detail as to why it failed.

A todo for now is fine.